stubs/arrayFunctions.stub

```diff
@@ -70,6 +70,18 @@ function array_udiff(
     callable $three
 ): int {}
 
+/**
+ * @template T
+ * @param array<T> $one
+ * @param array<T> $two
+ * @param callable(T, T): int $three
+ */
+function array_uintersect(
+    array $one,
+    array $two,
+    callable $three
+): int {}
```

Comment:
It'd be better to update the array_udiff and array_uintersect to actually use two different `@template` for each array (T + U).